src/commands/repos/WriteRepos.ts

```diff
@@ -64,7 +64,10 @@ export class WriteRepos extends AbstractReposCommand {
                 const status = {
                     url: current.url,
                     branch: currentBranch,
-                    description: current.description ? current.description : repoName
+                    description: current.description ? current.description : repoName,
+                    artifactGroup: current.artifactGroup,
+                    artifactVersion: current.artifactVersion,
+                    useSnapshot: current.useSnapshot
```

Comment:
Instead of listing the properties that we _do_ want to retain and override, would it be worthwhile to keep all by default, override the ones we want to change, and remove the ones we _do not_ want to retain? Using some `merge` functions and the magic of setting values to `undefined`, that should be easy. 🤔 

Both approaches have their drawbacks. The current one risks missing out on additional properties that we should keep. My suggestion risks keeping new properties that we should to get rid of. 🤷

Reply:
For now I'd prefer this explicit approach. Even if it duplicates stuff we can directly se in the code what is intended by each of the different variations. But I'm open for later refinements.

Reply:
Wouldn't this set the group to `undefined` of empty string if it was not set here from before?